coalib/parsing/FilterImplementation.py

```diff
@@ -0,0 +1,7 @@
+def LanguageFilter(languages, log_printer):
```

Comment:
so we could still do one filter one file, can't we? Actually the more you modularize, the better, maintaining multiple files is not a problem, it's even better than maintaining a single file ;)